TACC/tup-ui
# fix(tup-ui): change "User News" to "User Updates"

## Overview

Change "User News" to "User Updates".*

<sup>* Like was done in Core CMS and should be done in TAS.</sup>

## Related

- [New TUP Testing Session 2 - User Group Feedback](https://confluence.tacc.utexas.edu/display/UP/New+TUP+Testing+Session+2+-+User+Group+Feedback)

## Changes

- changed heading text

## Testing

1. Load Portal.
2. Confirm user updates are shown under the text "User Updates".

## UI

![user-updates-heading](https://user-images.githubusercontent.com/62723358/220405903-efc1534a-4fe2-4c9b-9f5c-d98f6ddcddc3.png)
